src/PullRequests.js

```diff
@@ -46,6 +47,22 @@ class PullRequests {
     }
   }
 
+  matchesAssignee(pr) {
+    if (this.assignee.values.length > 0) {
+      const result = _.some(this.assignee.values, (_assignee) => {
+        const matched = _assignee.match(/^.+\/(.+)$/)
+        const username = matched ? matched[1] : _assignee
+
+        return _.flatMap(pr.assignees.nodes, (request) => {
+          return request.login || request.name
```

Comment:
`request.name` is not accessible here because the GraphQL query doesn't specify `name` property at https://github.com/ohbarye/review-waiting-list-bot/pull/60/files#diff-ece4f699444a8c020a59acc9109100c7R88. 

I assume you can write just `return request.login` at this line.

FYI: `request.requestedReviewer` can have either `request.requestedReviewer.login` or `request.requestedReviewer.name` because the `requestedReviewer` is either a user or a team. But `pr.assignees` must be a user, so it never has `name`, I think.